domain/src/rdata/rfc1035.rs

```diff
@@ -32,6 +32,7 @@ use crate::base::serial::Serial;
 ///
 /// Implements some basic methods plus the `RecordData`, `FlatRecordData`,
 /// and `Display` traits.
+#[macro_export]
```

Comment:
Instead of exporting the macro here (because I think that makes it public), perhaps move it to `rdata::macros`? It’s the place for module-wide macros …

Reply:
Ah you're right, I'll move it to macros!